setup.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from setuptools import setup, find_packages
+
+
+setup(
+    name='FoxPuppet',
+    version='0.1.0',
+    description='Foxpuppet for Firefox UI testing',
+    long_description='See http://foxpuppet.readthedocs.org/',
+    url='https://github.com/mozilla/FoxPuppet',
+    license='MPL',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
+        'pytest',
+        'selenium'
```

Comment:
We should specify a minimum version of 3.0.1 (current), and we'll need to bump this when the 'context' context manager is in a released version of the client.